apps/src/MarkdownRenderer.js

```diff
@@ -0,0 +1,11 @@
+var marked = require('marked');
+
+var renderer = new marked.Renderer();
+renderer.html = function (html) {
+  return html.indexOf('<style>') !== -1 ? '' : html;
+};
+marked.setOptions({
+  renderer: renderer,
+});
+
+export default marked;
```

Comment:
Does this have the side effect of modifying `marked` globally when required? It might be better to have this file be called something like `StylelessRenderer.js` and to just export `renderer`, then for implementations to call `marked([string], { renderer })`